tests/serving/test_async_flow.py

```diff
@@ -1295,3 +1310,60 @@ def test_configure_model_runner_step_max_threads_processes(concurrency: str):
         ), "Max threads not configured properly"
     server.test(body={"n": 1})
     server.wait_for_completion()
+
+
+@pytest.mark.parametrize(
+    "model_class, raise_exception",
+    [
+        ("LLModel", True),
+        ("DummyAsyncLLMWithoutAsyncPredict", True),
+        ("DummyLLM", False),
+        ("DummyAsyncLLM", False),
+    ],
+)
+def test_llmodel_without_model_artifact(model_class, raise_exception):
+    #  LLModel should raise error because predict was not overridden
+    #  DummyAsyncLLMWithoutAsyncPredict should raise error because async_predict was not overridden
```

Comment:
```suggestion
@pytest.mark.parametrize(
    "model_class, raise_exception",
    [
        ("LLModel", True),  #  LLModel should raise error because predict was not overridden
        ("DummyAsyncLLMWithoutAsyncPredict", True), #  DummyAsyncLLMWithoutAsyncPredict should raise error because async_predict was not overridden
```
        ("DummyLLM", False),
        ("DummyAsyncLLM", False),
    ],
)
def test_llmodel_without_model_artifact(model_class, raise_exception):